src/actions/session.js

```diff
@@ -0,0 +1,31 @@
+import store from 'react-native-simple-store'
+import { setState } from '../actions/index'
+import { generateSessionID, fiveMinutesFromNow } from '../utils/session'
+
+export function storeSession(session) {
+  return () => {
+    store.save('@zooniverse:session', {
+      session
+    })
+  }
+}
+
+export function setSession() {
+  return dispatch => {
+    let stored = {}
+    store.get('@zooniverse:session').then(json => {
+      stored = json.session
+      if (stored.ttl < Date.now()) {
+        stored = generateSessionID()
```

Comment:
Should the `ttl` get reset here as well? It looks like on pfe we're only having the sessionID reset at this point, not creating an entirely new object.